src/main/scala-2.11/markets/settlement/SettlementMechanismActor.scala

```diff
@@ -15,5 +15,20 @@ limitations under the License.
 */
 package markets.settlement
 
+import markets.{Filled, StackableActor}
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1612110721/issues/source?bid=2921362&fileBranchId=3028966#l18)